Userland/Libraries/LibCore/File.cpp

```diff
@@ -85,8 +93,13 @@ int File::open_mode_to_options(OpenMode mode)
         flags |= O_EXCL;
     if (!has_flag(mode, OpenMode::KeepOnExec))
         flags |= O_CLOEXEC;
+#if !defined(AK_OS_WINDOWS)
     if (has_flag(mode, OpenMode::Nonblocking))
         flags |= O_NONBLOCK;
+#else
+    dbgln("Core::File::OpenMode::Nonblocking is not implemented");
```

Comment:
eh, I think this needs to be

```
    if (has_flag(mode, OpenMode::Nonblocking) {
#if !defined(AK_OS_WINDOWS)
        flags |= O_NONBLOCK;
#else
        dbgln("Core::File::OpenMode::Nonblocking is not implemented");
        VERIFY_NOT_REACHED();
#endif
    }
```

Otherwise any call to open_mode_to_options will crash.

Reply:
Sorry, fixed.